src/Components/Pages/Playlist/PlaylistPage.tsx

```diff
@@ -39,20 +42,53 @@ export default class PlaylistPage extends React.Component<PlaylistPageProps, Pla
         })
     }
 
+    onSearchSelect(playlist: SimplifiedPlaylistObject) {
+        const newPlaylist: PlaylistObject = {
```

Comment:
You can replace this with `const newPlaylist: PlaylistObject = { ...playlist, tracks: [] }` if you'd like.

Reply:
You are a life saver. I tried doing that like 47 different ways I couldn't get it to work